### PR TITLE
pgcleanup can redirect logging to stdout

### PR DESCRIPTION
**Pull request by gmauro**
this PR allows gxadmin to properly save logging details
https://github.com/usegalaxy-eu/gxadmin/blob/d3e98865bd9e3c509e3144b2f9974b03d41881e2/parts/25-galaxy.sh#L31

### CODE REVIEW

**Comment by jmchilton** [comment]
What do you think about a general log file command-line argument that can be sent "-" to push that to standard output? That seems a bit more standard to me - but I'm not certain.

**Comment by gmauro** [comment]
I am not sure about getting your point. Do you mean something like replacing the log-dir option with a log-file option and if log-file option is not set, the script send logs to stdout?

File: scripts/cleanup_datasets/pgcleanup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import galaxy.config
 from galaxy.exceptions import ObjectNotFound
 from galaxy.objectstore import build_object_store_from_config
-from galaxy.util.script import app_properties_from_args, populate_config_args
+from galaxy.util.script import app_properties_from_args, populate_config_args, set_log_handler
```
Comment (nsoranzo) [inline scripts/cleanup_datasets/pgcleanup.py]:
```suggestion
from galaxy.util.script import (
    app_properties_from_args,
    populate_config_args,
    set_log_handler,
)
```

File: lib/galaxy/util/script.py [hunk 2]
```diff
@@ -87,3 +88,11 @@ def _arg_parser():
     for argument in ARGUMENTS:
         parser.add_argument(*argument[0], **argument[1])
     return parser
+
+
+def set_log_handler(filename=None, stream=sys.stderr):
```
Comment (nsoranzo) [inline lib/galaxy/util/script.py]:
```suggestion
def set_log_handler(filename=None, stream=None):
```

**Comment by gmauro** [comment]
thanks @nsoranzo

**Review by nsoranzo — APPROVED** [review]

**Comment by mvdbeek** [comment]
Thank you @gmauro!

**Pull request merged**